src/Multiselect.vue

```diff
@@ -59,6 +61,7 @@
         :value="search"
         :disabled="disabled"
         :tabindex="tabindex"
+        :aria-label="name + '-searchbox'"
```

Comment:
Is this referencing an element or setting the label to a variable?
If it is referencing an element (`name + '-searchbox'` doesn't currently exist), it should be something like `:aria-labelled-by="name + '-searchbox'"` (see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby vs https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

Reply:
@bytesnz `aria-labelledby` is used when we want to reference any other element but as you have pointed out there no element we pointing to and just giving it an accessible name so we are using `aria-label` here.

Reply:
Just trying it out. It is probably worth not adding the attribute to this as people can do it themselves

```vue
<Multiselect :options="options" aria-label="Some select"></Multiselect>
```
works just fine.

I don't think it needs the `aria-owns` attribute either as the element it is pointing to is a descendant of the muliselect element.

If you haven't seen it already https://www.w3.org/WAI/ARIA/apg/patterns/combobox/ is a good reference.

Reply:
@bytesnz Thanks for looking into this PR. For now we are just going ahead with this, if you have any enhancements in the mind then please open a PR for the same.